server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CachedBlobStore.java

```diff
@@ -154,14 +149,50 @@ public BucketName getDefaultBucketName() {
         }
     }
 
-    /**
-     * bytes: byte[] from PushbackInputStream.If PushbackInputStream is empty bytes.length == 1
-     */
+    private Publisher<BlobId> saveInCache(BucketName bucketName, InputStream inputStream, StoragePolicy storagePolicy) {
+        PushbackInputStream pushbackInputStream = new PushbackInputStream(inputStream, sizeThresholdInBytes + 1);
```

Comment:
still a +1

Reply:
Can we adopt the same style than here: #3295 (comment)

Reply:
Yes, i doing it